httpbin/httpbin.go

```diff
@@ -97,76 +124,59 @@ var _ http.Handler = &HTTPBin{}
 // Handler returns an http.Handler that exposes all HTTPBin endpoints
 func (h *HTTPBin) Handler() http.Handler {
 	mux := http.NewServeMux()
-
 	mux.HandleFunc("/", methods(h.Index, "GET"))
-	mux.HandleFunc("/forms/post", methods(h.FormsPost, "GET"))
-	mux.HandleFunc("/encoding/utf8", methods(h.UTF8, "GET"))
-
-	mux.HandleFunc("/delete", methods(h.RequestWithBody, "DELETE"))
-	mux.HandleFunc("/get", methods(h.Get, "GET"))
-	mux.HandleFunc("/head", methods(h.Get, "HEAD"))
-	mux.HandleFunc("/patch", methods(h.RequestWithBody, "PATCH"))
-	mux.HandleFunc("/post", methods(h.RequestWithBody, "POST"))
-	mux.HandleFunc("/put", methods(h.RequestWithBody, "PUT"))
-
+	mux.HandleFunc("/absolute-redirect/", h.AbsoluteRedirect)
 	mux.HandleFunc("/anything", h.Anything)
 	mux.HandleFunc("/anything/", h.Anything)
-
-	mux.HandleFunc("/ip", h.IP)
-	mux.HandleFunc("/user-agent", h.UserAgent)
-	mux.HandleFunc("/headers", h.Headers)
-	mux.HandleFunc("/response-headers", h.ResponseHeaders)
-	mux.HandleFunc("/hostname", h.Hostname)
```

Comment:
Aside from adding the new `/sse` endpoint, this is just consolidating and alphabetizing the entries